crates/bin/pindexer/src/block.rs

```diff
@@ -38,7 +38,7 @@ CREATE TABLE IF NOT EXISTS block_details (
         event: &ContextualizedEvent,
     ) -> Result<(), anyhow::Error> {
         let pe = pb::EventBlockRoot::from_event(event.as_ref())?;
-        let timestamp = pe.timestamp.expect("Block has no timestamp");
+        let timestamp = pe.timestamp.unwrap_or_default();
```

Comment:
The solution to this is to use the version of pd on the main for indexing events, we don't want to silently have January 1970 timestamps, we want to fail early when we're using the wrong version of pd because other kinds of event omission are worse.

Reply:
No, this was on purpose in order to make it work.